src/main/java/edu/harvard/iq/dataverse/export/ExportService.java

```diff
@@ -51,9 +57,50 @@ public class ExportService {
 
     private static ExportService service;
     private ServiceLoader<Exporter> loader;
+    private Map<String, Exporter> exporterMap = new HashMap<String, Exporter>();
+
+    private static final Logger logger = Logger.getLogger(ExportService.class.getCanonicalName());
 
     private ExportService() {
-        loader = ServiceLoader.load(Exporter.class);
+        /*
+         * Step 1 - find the EXPORTERS dir and add all jar files there to a class loader
+         */
+        List<URL> jarUrls = new ArrayList<URL>();
+        Path exporterDir = Paths.get(JvmSettings.EXPORTERS_DIRECTORY.lookup());
```

Comment:
Just a thought: maybe make this lookup optional and skip if not present. Someone not using the plugin function might not want to see warnings about no directory for exporters.

If we still want a sane default location in `microprofile-config.properties`, maybe add an enabling config property?